_includes/topnav.html

```diff
@@ -2,84 +2,82 @@
 
 <div class="top-nav">
 
-	<img src="{{ "images/weebly_cloud_logo.svg" }}" alt="Weebly Cloud"/>
+	<div class="TopNavItem">
```

Comment:
To keep things consistent with Jekyll world, let's use kebab-case for new class names (`.top-nav-item`). This will also make it easier to spot anything is utilizing styles from Orbit since those will be in TitleCase.